src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -294,33 +302,22 @@ private void addIssueProperties(String issueKey, String provider, LinkableItem t
         );
     }
 
-    private String createOperationComment(ItemOperation operation, String category, String provider, Collection<ComponentItem> componentItems) {
+    private List<String> createOperationComment(ItemOperation operation, String category, String provider, Collection<ComponentItem> componentItems) {
         JiraIssueFormatHelper jiraChannelFormatHelper = new JiraIssueFormatHelper();
-        String attributesString = jiraChannelFormatHelper.createComponentAttributesString(componentItems);
-
-        StringBuilder commentBuilder = new StringBuilder();
-        commentBuilder.append("The ");
-        commentBuilder.append(operation.name());
-        commentBuilder.append(" operation was performed for this ");
-        commentBuilder.append(category);
-        commentBuilder.append(" in ");
-        commentBuilder.append(provider);
-        if (StringUtils.isNotBlank(attributesString)) {
-            commentBuilder.append(".\n----------\n");
-            commentBuilder.append(attributesString);
-        } else {
-            commentBuilder.append(".");
-        }
-        return commentBuilder.toString();
+        return jiraChannelFormatHelper.createOperationComment(operation, category, provider, componentItems);
     }
 
-    private IssueRequestModelFieldsBuilder createFieldsBuilder(ComponentItem arbitraryItem, Collection<ComponentItem> componentItems, LinkableItem commonTopic, Optional<LinkableItem> subTopic, String provider) {
+    private IssueContentModel createContentModel(ComponentItem arbitraryItem, Collection<ComponentItem> componentItems, LinkableItem commonTopic, Optional<LinkableItem> subTopic, String provider) {
         final JiraIssueFormatHelper jiraChannelFormatHelper = new JiraIssueFormatHelper();
-        final IssueRequestModelFieldsBuilder fieldsBuilder = new IssueRequestModelFieldsBuilder();
         final String title = jiraChannelFormatHelper.createTitle(provider, commonTopic, subTopic, arbitraryItem.getComponentKeys());
-        final String description = jiraChannelFormatHelper.createDescription(commonTopic, subTopic, componentItems, provider);
-        fieldsBuilder.setSummary(title);
-        fieldsBuilder.setDescription(description);
+        final IssueContentModel descriptionModel = jiraChannelFormatHelper.createDescription(commonTopic, subTopic, componentItems, provider);
+        return IssueContentModel.of(title, descriptionModel.getDescription(), descriptionModel.getAdditionalComments());
```

Comment:
This seems slightly redundant with the line right above it. Could we have the createDescription just return the proper model with the title added?